bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/ReadLacProcessorV3.java

```diff
@@ -89,10 +89,10 @@ private ReadLacResponse getReadLacResponse() {
             }
         } catch (Bookie.NoLedgerException e) {
             status = StatusCode.ENOLEDGER;
-            logger.error("No ledger found while trying to read last entry: {}", ledgerId, e);
+            logger.warn("No ledger found while trying to read last entry: {}", ledgerId, e);
         } catch (IOException e) {
             status = StatusCode.EIO;
-            logger.error("IOException while trying to read last entry: {}", ledgerId, e);
+            logger.warn("IOException while trying to read last entry: {}", ledgerId, e);
```

Comment:
We should only change the logger level to WARN for `NoEntryException` not all  IOException. The purpose of this PR should be if users provided a wrong ledger ID or an entry ID, the log level should be WARN.